ide/spi.debugger.ui/src/org/netbeans/modules/debugger/resources/mf-layer.xml

```diff
@@ -330,6 +330,11 @@
         </folder>
 
         <folder name="Modes">
+            <folder name="navigator">
+                <file name="breakpointsView.wstcref" url="modes/navigator/breakpoints.wstcref">
+                    <attr name="position" intvalue="400"/>
+                </file>
+            </folder>
```

Comment:
nitpick: not sure if the registration order is relevant, but consider moving this down below the evaluator folder, so that the registration order resembles somewhat the order of appearance - makes it easier to read for me at least.

Reply:
The order is completely irrelevant. There is even `output` between the folders. Anyway, it is no harm to 9b461e1335